rubocop/rubocop_rails.yml

```diff
@@ -3,11 +3,34 @@ inherit_from: rubocop.yml
 AllCops:
   Exclude:
     - db/schema.rb
+    - 'bin/**/*'
+    - 'config/**/*'
     - 'vendor/**/*'
 
 Rails:
   Enabled: true
 
+Metrics/BlockLength:
+  Exclude:
+    - 'db/seeds.rb'
+    - 'spec/**/*'
+
+Metrics/LineLength:
+  Exclude:
+    - 'db/seeds.rb'
+
 Metrics/MethodLength:
   Exclude:
     - 'db/migrate/**/*'
+
+Metrics/ModuleLength:
+  Exclude:
+    - 'db/seeds.rb'
+    - 'spec/**/*'
+
+Style/StringLiterals:
+  Exclude:
+    - 'config.ru'
+    - 'Gemfile'
```

Comment:
The `Gemfile` is excluded from the double-quotes cop in our [base](https://github.com/TheGnarCo/gnar-style/blob/86e5b8c14efa8d4601c532f062740850254ce365/rubocop/rubocop.yml#L4) cop already.